src/plugins/discord/createGraph.js

```diff
@@ -323,6 +323,12 @@ export function _createGraphFromMessages(
       wg.graph.addEdge(authorsMessageEdge(message, author));
     }
 
+    const roleMultipliedReactingMembers = Array.from(
```

Comment:
would love to hear your opinion on this but should this code be always executed when sometimes it won't be necessary?

Reply:
fair point, we could short circuit it by checking the flag. Might as well.